pages/_/stats.js

```diff
@@ -0,0 +1,414 @@
+import * as React from "react";
+import * as Strings from "~/common/strings";
+import * as Constants from "~/common/constants";
+import * as SVG from "~/components/system/svg";
+import * as System from "~/components/system";
+import * as SchemaTable from "~/common/schema-table";
+import Section from "~/components/core/Section";
```

Comment:
All the other components in the codebase separate these sections with a white space, As you are making these adjustments please make sure you do it on other files too.

Reply:
All set.